components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/http/session.ts

```diff
@@ -16,56 +16,58 @@
 
 const HttpSessionFacade = Java.type("org.eclipse.dirigible.components.api.http.HttpSessionFacade");
 
-export function isValid() {
-	return HttpSessionFacade.isValid();
-};
+export class Session {
+	public static isValid(): boolean {
+		return HttpSessionFacade.isValid();
+	};
 
-export function getAttribute(name) {
-	return HttpSessionFacade.getAttribute(name);
-};
+	public static getAttribute(name: string): string {
+		return HttpSessionFacade.getAttribute(name);
+	};
 
-export function getAttributeNames() {
-	const attrNames = HttpSessionFacade.getAttributeNamesJson();
-	if (attrNames) {
-		return JSON.parse(attrNames);
-	}
-	return attrNames;
-};
+	public static getAttributeNames(): Array<string> {
```

Comment:
It seems that there is no difference between `string[]` and `Array<string>` according to this: https://stackoverflow.com/questions/36842158/arraytype-vs-type-in-typescript

Please be consistent in using either `string[]` or `Array<string>`.

> _**Side Comment**: For me `string[]` seems more natural for the JS/TS context, where `Array<string>` seem more appropriate for the Java context._